src/auth_token.c

```diff
@@ -38,9 +38,14 @@
 /*                            File Scoped Variables                           */
 /*----------------------------------------------------------------------------*/
 void createCurlheader(char *mac_header, char *serial_header, char *uuid_header, char *transaction_uuid, struct curl_slist *list, struct curl_slist **header_list);
+long g_response_code;
```

Comment:
Is global variable needed, as its used within the same function

Reply:
Yes, global variable is added for auth token unit tests where it needs to check response code through libcurl mock functions.